_drgn.pyi

```diff
@@ -1953,6 +1959,51 @@ class TypeKind(enum.Enum):
     FUNCTION = ...
     """Function type."""
 
+class SymbolBinding(enum.Enum):
+    "A ``SymbolBinding`` represents the linker visibility of a symbol."
+
+    UNKNOWN = ...
+    """The binding type is unknown."""
+
+    LOCAL = ...
+    """The symbol is local to its module."""
+
+    GLOBAL = ...
+    """The symbol is global."""
+
+    WEAK = ...
+    """The symbol is weak and may be overridden."""
+
+    UNIQUE = ...
+    """The symbol is unique among all global and local symbols."""
+
+class SymbolKind(enum.Enum):
+    "A ``SymbolKind`` represents what sort of value a symbol represents."
```

Comment:
I waffled over the word choice. You're right, value is typically used differently in the context of programming/debugging. I didn't want to use Object due to the overloading, since really only one of these (DATA) would be an object in the technical sense - others would be functions or something else entirely. Maybe "entity" works? `A SymbolKind describes what sort of entity a symbol represents`
(also getting rid of the double "represents"...)

Reply:
Resolving this, feel free to reopen if you disagree with entity.